changelog.d/2068.added.md

```diff
@@ -0,0 +1,9 @@
+Add fs mapping, under `feature.fs.mapping` now it's possible to specify regex match and replace for paths while running mirrord exec.
+
+Example:
+
+```toml
+[feature.fs.mapping]
+"/var/app/temp" = "/tmp" # Will replace all calls to read/write/scan for "/var/app/temp/sample.txt" to "/tmp/sample.txt"
+"/var/app/.cache" = "/workspace/mirrord$0" # Will replace "/var/app/.cache/sample.txt" to "/workspace/mirrord/var/app/.cache/sample.txt" see [Regex::replace](https://docs.rs/regex/latest/regex/struct.Regex.html#method.replace)
```

Comment:
I'd like users to have the ability to group different parts of the path
for example
`"/(var)/(app)/.cache" = "/$0/bbb/$1"`
is it possible?

Reply:
yep, capture groups work even named ones like "/(?<base>var)/(?<sub>app)/.cache"  = "/.cache/$base-$sub"